### DISCUSSION TITLE
styling table headings

### DISCUSSION THREAD

**Discussion by feorlen** [pradyunsg/furo#267] [Ideas] [answered]
I'd like to change the colors for table headings and have it work with light/dark mode. There isn't a lot of contrast with the current colors, and some of my readers have noted they sometimes scan the page and don't see a table amid the other text.

The table heading background uses `color-background-secondary`, which is easily configurable. But a lot of other things use it as well. I can do it with `table.docutils thead th` but that makes it the same color all the time rather than change with light/dark.

Is there a way to allow styling these elements that doesn't impact other things? Thanks

<img width="717" alt="light text on nearly black gray, with black page background." src="https://user-images.githubusercontent.com/95028/134950410-fd20d479-ba4d-4c4e-8c2c-d686ae5a0098.png">

<img width="738" alt="dark text on light gray, with white page background." src="https://user-images.githubusercontent.com/95028/134950416-2da76f6d-8f7c-4139-be9d-411f1a93af48.png">
!

**Comment by pradyunsg** [comment, marked as answer]
Done in 5cdb35c0 -- I'll include that in the next release, which is likely gonna be next weekend or the weekend after. :)

**Reply by feorlen** [reply]
That's awesome, thank you!
